pyproject.toml

```diff
@@ -34,7 +34,9 @@ classifiers = [
     'Operating System :: MacOS',
     'Typing :: Typed',
 ]
-dependencies = ['typing-extensions>=4.6.0,!=4.7.0']
+dependencies = [
+    'typing-extensions>=4.12.0',
```

Comment:
Do we need to call this out in the changelog?

Reply:
Pydantic already requires 4.12, so I don't think so.